packages/trip-form/src/SliderSelector/index.tsx

```diff
@@ -0,0 +1,95 @@
+import CSS from "csstype";
+import React, { ChangeEvent, ReactElement, useCallback } from "react";
+
+import * as S from "../styled";
+
+// eslint-disable-next-line prettier/prettier
+import type { QueryParamChangeEvent } from "../types";
+
+interface SliderSelectorProps {
+  /**
+   * The CSS class name(s) to apply to this element.
+   */
+  className?: string;
+  /**
+   * The contents of the contained <label> control.
+   */
+  label?: ReactElement | string;
+  /**
+   * A unique name for the setting.
+   */
+  name?: string;
+  /**
+   * Triggered when the value of the <input> control changes.
+   */
+  onChange?: (evt: QueryParamChangeEvent) => void;
+  /**
+   * Standard React inline style prop.
+   */
+  style?: CSS.Properties;
+  /**
+   * The initial min value for the contained <input> control.
+   */
+  min: number;
+  /**
+   * The initial max value for the contained <input> control.
+   */
+  max: number;
```

Comment:
Sort props.

Reply:
Sorted in 6b990699ed66451246a3bb0c3c75e37763c123f6